docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -118,9 +134,9 @@ drwxr-xr-x 3 root root   4096 Jul 21 19:46 ..
 -rw-r--r-- 1 root root   1292 Jul 21 19:47 wordpress.sql
 {{< /output >}}
 
-You should also have one snapshot in your Restic repository:
+You should also have one snapshot in your Restic repository; use Restic's `snapshot` command to view it:
 
-    restic -r s3:us-east-1.linodeobjects.com/your-bucket-name -p /root/restic_pw snapshots
+    sudo /bin/bash -c "source /root/restic_params; restic -r s3:us-east-1.linodeobjects.com/your-bucket-name -p /root/restic_pw snapshots"
```

Comment:
A few issues here that popped up later in the guide too:
- The original restic command was not successful without the object storage creds in the `/root/restic_params` file
- Because `/root/restic_params` and `/root/restic_pw` are owned by root, this needed sudo
- Because the `source` command is used, I ended up needing to create an inline bash script that ran that and then ran restic. There might be another way to do this that is more elegant, but I couldn't think of any other options

Reply:
> There might be another way to do this that is more elegant, but I couldn't think of any other options

If “source /root/restic_params“ is in the root user’s Bash profile script, does “sudo -i ...” bring these into the environment in which it runs Restic?